web/stories/pages/posts/fast-scene.svelte

```diff
@@ -0,0 +1,21 @@
+<script lang="ts">
```

Comment:
Can we avoid special `FastScene` and `SlowScene` and just use `Scene`?

We will need to set posts in many places.

Reply:
I'm kinda afraid that `Scene` become God component.
Maybe it's better to keep `FastScene` and `SlowScene` but only contain a slot in them?

Reply:
The God component is sort of a plan here.

I understand that it is a bad practice for general component. But what is the problem for mocking component?